dir.props

```diff
@@ -76,11 +76,13 @@
 
   <!-- Output paths -->
   <PropertyGroup>
-    <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)' == ''">$(RootBinDir)obj</BaseIntermediateOutputPath>
+    <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)' == ''">$(RootBinDir)obj/</BaseIntermediateOutputPath>
```

Comment:
Inside BuildVersion.targets(imported bellow), we define that the location of _version.h and the generated BuildVersion.props file will be $(BaseIntermediateOutputPath)_version.h, so we assume that BaseIntermediateOutputPath ends with a slash. That might be an error in buildtools itself, but since it has no impact to make this small change, it was easier to add the slash here than to have to publish a new version of buildtools.